examples/cli.c

```diff
@@ -124,30 +124,24 @@ input_cmd(void)
 				return cmd;
 		}
 	}
-cleanup:
+ cleanup:
 	free(cmd);
 	return NULL;
 }
 
-static const char *
-help(int argc, char *argv[])
+static const char *help(int argc, char *argv[])
 {
 	return
-"available commands:\n"
-"\n"
-"help\n"
-"set <option> <value> ...\n"
-"subset <section> <option> <value>\n"
-"create <section>\n"
-"destroy <section>\n"
-"dump\n"
-"quit\n"
-"\n"
-"<option> is one of 'bool', 'int' 'string' and 'float'.\n";
+	    "available commands:\n"
+	    "\n"
+	    "help\n"
+	    "set <option> <value> ...\n"
+	    "subset <section> <option> <value>\n"
+	    "create <section>\n"
+	    "destroy <section>\n" "dump\n" "quit\n" "\n" "<option> is one of 'bool', 'int' 'string' and 'float'.\n";
```

Comment:
The script is only a help, I went over all files and checked manually afterwards.  I don't agree with you on this particular issue, since the text is in a `return` statement it should be indented,